### DISCUSSION TITLE
Edit Variable and Right Click

### DISCUSSION THREAD

**Discussion by Dhuls-Scratch** [ScratchAddons/ScratchAddons#3471] [Ideas]
- Replace the "Rename Variable" and "Edit List" options with "Edit Variable" and "Edit List" options that allow you to
    - Toggle whether the variable is cloud (variables only)
    - Rename the variable or list
    - Toggle whether or not the variable or list is "For all sprites" or "For this sprite only" (will not work with the stage)
- When right clicking a variable or list reporter block, an "Edit/Rename variable/list" button should appear

**Comment by Secret-chest** [comment]
Added
